### ISSUE TITLE
Ceedling v0.29.0 version cmd

### ISSUE THREAD

**Issue by JuPrgn** [ThrowTheSwitch/Ceedling#415]
I noticed that calling `ceedling version` with v0.29.0 returns : 

```
C:\Users\Julien>ceedling version
Welcome to Ceedling!
  Ceedling::   0.29.0
  CMock::      0.0.0
  Unity::      0.0.0
  CException:: 0.0.0
```

Or : 

```
C:\Users\Julien\Documents\TestProject>ruby vendor\ceedling\bin\ceedling version
  Ceedling:: 0.29.0
CException:: UNKNOWN.UNKNOWN
     CMock:: UNKNOWN.UNKNOWN
     Unity:: UNKNOWN.UNKNOWN
```

**Comment by mvandervoord** [comment]
Hi.

Was TestProject an existing project or is it a new project after installing the latest gem? If it's an existing project and it runs local ceedling (which it appears to, given you're checking the vendor directory), have you run `ceedling upgrade` on the project to update it to the latest tools?

**Comment by JuPrgn** [comment]
This is a new project and it runs local ceedling.

**Comment by pmembrey** [comment]
I installed `0.28.0` the day before `0.29.0` was released. When I upgraded the gem, I tried creating a new project with `ceedling new --local MyProj`. This gave the same `UNKNOWN` error as above. However, upgrading the original project from the day before, I am able to see version numbers (though I don't know if these are correct):

      Ceedling:: 0.29.0
    CException:: 1.3.1.18
         CMock:: 2.4.6.217
         Unity:: 2.4.3.122

I get the same `UNKNOWN` is with either of these commands:

`ceedling new --local MyProj`

`ceedling new MyProj2`
